dbt_coves/tasks/generate/base.py

```diff
@@ -156,3 +166,194 @@ def get_metadata_columns(self, relation, cols):
                 new_col = self.get_default_metadata_item(col.name, type=col.dtype)
             metadata_cols.append(new_col)
         return metadata_cols
+
+    def render_property_files(
+        self,
+        context,
+        options,
+        templates_folder,
+        update_strategy,
+        object,
+        destination=None,
+        template=None,
+    ):
+        strategy_key_update_all = ""
+        strategy_key_recreate_all = ""
+        if object == "Models":
+            yml_cfg_destination = destination
+            # If destination does not contain Jinja syntax on last component it's a single file,
+            # and update/recreate must behave as 'update all' or 'recreate all'
+            if not re.search(
+                r"\{\{[A-Za-z]*\}\}", yml_cfg_destination.split("/")[-1].replace(" ", "")
```

Comment:
This is working only in linux environments. Why you need to set "model_prop_is_single_file" = True?